bitnami/rabbitmq/values.yaml

```diff
@@ -95,6 +95,10 @@ diagnosticMode:
 hostAliases: []
 ## @param commonAnnotations Annotations to add to all deployed objects
 ##
+dnsConfig: {}
+## @param dnsConfig to modify resolv.conf for pod
+## https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/
+##
 commonAnnotations: {}
```

Comment:
```suggestion
## @param commonAnnotations Annotations to add to all deployed objects
##
commonAnnotations: {}
## @param dnsConfig Used to modify resolv.conf for pod
## https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/
##
dnsConfig: {}
```

Reply:
Hi @carrodher,
Version bumped.
Thanks,

Reply:
Please, take also a look at this suggestion ⬆️